deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -27,27 +27,29 @@ def __init__(self,
                n_pair_feat=14,
                n_hidden=50,
                n_graph_feat=128,
+               mode="classification",
```

Comment:
Here and a few other places, you added `mode` as an argument but didn't add it to the docstring.